src/ostorlab/agent/message/proto/v3/report/cve/cve_pb2.py

```diff
@@ -13,27 +13,1206 @@
+)
+
+
+
+
+_MESSAGE = _descriptor.Descriptor(
+  name='Message',
+  full_name='ostorlab.agent.message.proto.v3.report.cve.Message',
+  filename=None,
+  file=DESCRIPTOR,
+  containing_type=None,
+  create_key=_descriptor._internal_create_key,
+  fields=[
+    _descriptor.FieldDescriptor(
+      name='cve_id', full_name='ostorlab.agent.message.proto.v3.report.cve.Message.cve_id', index=0,
+      number=1, type=9, cpp_type=9, label=1,
+      has_default_value=False, default_value=b"".decode('utf-8'),
+      message_type=None, enum_type=None, containing_type=None,
+      is_extension=False, extension_scope=None,
+      serialized_options=None, file=DESCRIPTOR,  create_key=_descriptor._internal_create_key),
```

Comment:
please double check which protoc version is used.. make sure this file is compact..